guides/rules/no-timers.md

```diff
@@ -1,2 +1,27 @@
 # No Timers
 
+**TL;DR: You should avoid using timers in favor of Ember.later**
+
+In general, it's best to avoid the use of `setTimeout` and `setInterval` in Ember. 
```

Comment:
Yes. There is are legit cases to use these but they are far and in between. Best that we tell people to avoid them and just escape valve them with `eslint:ignore`.